NoahSaxophone.md

```diff
@@ -0,0 +1,25 @@
+---
+name: "@NoahSaxophone"
+project: "ADS-B Receiver"
+---
+
+# ADS-B Receiver
+
+## Summary
+
+I am going to build an ADS-B receiver so I can track a plane's location in the sky. Right now, my plan is to become a pilot when I turn 17. I've joined the Royal Canadian Air Cadets and hope to get my license through them. I've wanted to do this project for a long time now, but I couldn't find a raspberry pi that was cheap enough to do so.
+
+## Plan
+
+After purchasing the components, my plan is to put them together. Next, I will download FlightRadar's Pi24 software and put it on the raspberry pi. Then, I will set it up and (hopefully) be able to see each plane's telemetry. Finally, I will design and 3-D print a case for everything to allow it to be outside. I know it sounds simple, but I find it fascinating that anyone can just scan the sky for planes.
+
+## Budget
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Raspberry Pi 3 Model B+   | https://www.amazon.ca/Raspberry-Pi-3-Model-B/dp/B00LPESRUK/ref=sr_1_3?crid=181W7X9K1ZD7 | $119.36 CAD ($88.26 USD) |
```

Comment:
This link appears not to work for me. Does it work for you?

<img width="1552" alt="Screenshot 2022-12-29 at 10 43 11 AM" src="https://user-images.githubusercontent.com/72365100/209976473-68899011-d81e-41f5-a499-336bb4fa0e5f.png">

Do you also need an SD card for your pi?

Reply:
That's weird because the link works for me. 

![image](https://user-images.githubusercontent.com/121519033/209983575-37871b2e-9578-4206-898f-b0483b20b503.png)

Also, the price in the picture is different from the one in the budget. However, when I go to checkout on amazon, the price goes back to the price on the budget. I think this is because, on the product screen, it says import fees are included, but when I go to checkout, they are separate.

I already have an SD card for the pi.

Reply:
Great, thanks for clarifying.